pyiron/atomistics/structure/atoms.py

```diff
@@ -1710,6 +1719,31 @@ def plot3d(
 
         view.camera = camera
         view.background = background
+
+        if camera_axis not in ['x', 'y', 'z']:
+            raise ValueError(
+                "Choose between x, y, or z"
+            )
+
+        if camera_zoom_out is None:
+            camera_zoom_out = 14
```

Comment:
I guess you can put this as default value (i.e. `plot3d(self, ..., camear_zoom_out=14, ...)`)

Reply:
Updated.